NuRadioReco/detector/detector.py

```diff
@@ -57,7 +57,7 @@ def Detector(*args, **kwargs):
             - kwargs["source"] == "rnog_mongo" -> `NuRadioReco.detector.RNO_G.rnog_detector`
             - kwargs["source"] == "sql" -> `NuRadioReco.detector.detector_base`
             - kwargs["source"] == "json" or "dictionary" -> `NuRadioReco.detector.detector_base` or
-              `NuRadioReco.detector.generic_detector`
+            `NuRadioReco.detector.generic_detector`
```

Comment:
bad rst formatting (previous was correct)

Reply:
okay, I changed it because my editor complained ...